### ISSUE TITLE
Migrate to pipenv to manage Python dependencies

### ISSUE THREAD

**Issue by SaxyPandaBear** [SaxyPandaBear/discord-food-bot#23]
see title

**Comment by Pretty-19** [comment]
@SaxyPandaBear  was interested in this. But is this a good first issue for someone with basic Python skills?

**Comment by SaxyPandaBear** [comment]
Yeah it's pretty straightforward, but isn't really a lot of Python coding. It's more setup focused. Take a look at pipenv [docs](https://pipenv.pypa.io/en/latest/) and see what you can understand from that. All you really need to do is introduce a Pipfile that pulls in the dependencies that are needed to run the project, which are discord.py, PRAW, and redis-py. 

This is deployed on Heroku, so also need to make sure that Heroku accepts reading a pipfile for dependencies instead of a requirements.txt file

**Comment by Pretty-19** [comment]
@SaxyPandaBear  Thanks  will have a check on that

**Comment by Pretty-19** [comment]
Pipenv is giving me could not find a version that matches  botocore==1.12.253

**Comment by SaxyPandaBear** [comment]
What are you pulling in with botocore? If you look at what's in the requirements.txt, those are the only things (and their dependencies) that are needed for this project. None of those things depend on boto3.

What are the contents of your Pipfile?

**Comment by Pretty-19** [comment]
Yes the requirements file is containing other dependencies will remove and try

**Comment by Pretty-19** [comment]
1. was able to create pipfile.lock and pipenv file ( removed requirements.txt) .
2. while deploying to heroku saw a warning and application gave me an error

![image](https://user-images.githubusercontent.com/42793632/114655383-bdd43680-9d09-11eb-98f9-4b62cdb58648.png)
![image](https://user-images.githubusercontent.com/42793632/114655497-f116c580-9d09-11eb-8c2e-b60e6ae0fbb8.png)

**Comment by SaxyPandaBear** [comment]
Are you using your own credentials for Discord and Reddit? I'm not giving mine out. Those are configured in Heroku as secrets to be read.

**Comment by SaxyPandaBear** [comment]
If you just submit a PR, I should be able to tell if it'll work.

**Comment by Pretty-19** [comment]
Raised a PR can you confirm if this works for you ?
